data/service_accounts/domain.yml

```diff
@@ -47,16 +48,65 @@ slots:
         intent: intent_core_no
         conditions: *need_another_integration
 
+  new_or_existing:
```

Comment:
We've been scoping these var and intents to avoid conflicts, like service_account_new_or_existing.

The names do get kind of long, maybe could abbreviate for all of these. Either way I think it'd be helpful, could do in another PR if thatd be easier.

Reply:
Just renamed this slot only. I other ones have `service_account` in it already.